assets/cms/scss/_variables.scss

```diff
@@ -151,3 +151,10 @@ $panel-form-element-bg-color: #4c4e53;
 $panel-form-element-text-color: #bababa;
 $panel-normal-text-color: #676767;
 $panel-datepicker-highlight-day-color: #828282;
+
+/* Sitemap Icons */
```

Comment:
This looks good - can you add 'sitemap-' before all the variable names so it's clear that that's where this is being used?

Reply:
Just updated the variable names